File: Jaewon/Level.1/7) 3진수/7.cpp
#include <iostream>
#include <string>
#include <vector>

using namespace std;

int solution(int n) {
    int answer = 0;
    string result = "";
    
    while (n > 0){
        int tempInt = (n % 3);
        char tempChar = static_cast<char>(tempInt);
        result += tempChar;
        n /= 3;
    }

    int count = 1;
    
    for(int i = result.size()-1; i >= 0; i--){
        answer += result[i] * count;
        count *= 3;
    } 
    
    return answer;
}

File: Jaewon/Level.1/7) 3진수/7.py
def solution(n):
    threeNum = ''
    while n > 0:
        threeNum += str(n%3)
        n = n//3
    answer = 0
    count = 1

    for eachChar in reversed(threeNum):
        answer += count * int(eachChar)
        count *= 3

    return answer